data/ignition/files/usr/local/bin/common.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+get_host() {
```

Comment:
This is copied from start-assisted-service, but the other copy has not been removed. In any event, I don't think start-assisted-service needs to exist any more because the host IP is set by directly templating the configmap.